### PR TITLE
chore(web): Updated Sýslumenn's email list

### PR DESCRIPTION
**Pull request by RunarVestmann**
# Updated Sýslumenn's email list

## What

* Updated the email list for when someone contacts Sýslumenn from the service web
* Made the code for the email list more readable by replacing id's with variables

## Why

* This was a requested feature

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts [hunk 1]
```diff
@@ -4,55 +4,55 @@ import { environment } from '../environments/environment'
...
+  | typeof ANDLAT_OG_DANARBU
+  | typeof THINGLYSINGAR
+  | typeof GJOLD_OG_INNHEIMTA
+  | typeof LOGGILDINGAR
+  | typeof VOTTORD
+  | typeof LOGRADAMAL
+  | typeof ONNUR_THJONUSTA_SYSLUMANNA
+  | typeof LEYFI
+  | typeof FULLNUSTUGERDIR
+  | typeof DEFAULT
```
Comment (kristofer-lp8f) [inline libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts:55]:
```suggestion
  | string
```
This doesn't really make sense to me. The types are all `string`.

File: libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts [hunk 2]
```diff
@@ -4,55 +4,55 @@ import { environment } from '../environments/environment'
...
+
+// Leyfi
+const LEYFI = '7HbSNTUHJReJ2GPeT1ni1C'
+
+// Fullnustugerðir
+const FULLNUSTUGERDIR = '7LkzuYSzqwM7k8fJyeRbm6'
+
+// Default/fallback email address
+const DEFAULT = 'default'
+
```
Comment (kristofer-lp8f) [inline libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts:42]: Have you considered using enum?

**Review by kristofer-lp8f — APPROVED** [review]
LGTM 👍 

I'd recommend refactoring to use enums instead of copy-pasting/retyping IDs. The change you are doing is for the better, nonetheless :)

File: libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts (hunk 1 above)
Comment (RunarVestmann) [reply libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts:55]: Aren't the types the actual possible value, like typeof FJOLSKYLDUMAL is the id, right?

File: libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts (hunk 2 above)
Comment (RunarVestmann) [reply libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts:42]: I've never used enums in TypeScript before but it'd probably be a smarter move: https://www.typescriptlang.org/docs/handbook/enums.html

Comment (RunarVestmann) [reply libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts:42]: image
This is way better :)

Comment (kristofer-lp8f) [reply libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts:42]: @RunarVestmann

Comment (RunarVestmann) [reply libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts:42]: I addressed this in another PR: https://github.com/island-is/island.is/pull/7410

**Pull request merged**